tf-101/2-experiencing-terraform.md

```diff
@@ -174,241 +198,260 @@ should now work.
-      + default_dhcp_options_id  = (known after apply)
-      + default_route_table_id   = (known after apply)
-      + default_security_list_id = (known after apply)
-      + defined_tags             = (known after apply)
-      + display_name             = "tf-101"
-      + dns_label                = "tf101"
-      + freeform_tags            = (known after apply)
-      + id                       = (known after apply)
-      + ipv6cidr_blocks          = (known after apply)
-      + is_ipv6enabled           = (known after apply)
-      + state                    = (known after apply)
-      + time_created             = (known after apply)
-      + vcn_domain_name          = (known after apply)
-    }
-
-Plan: 2 to add, 0 to change, 0 to destroy.
-
-Changes to Outputs:
-  + vcn_state = (known after apply)
-
-───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
-
-Note: You didn't use the -out option to save this plan, so Terraform can't guarantee to take exactly these actions if you run "terraform apply"
-now.
-$ 
-```
-
-What we're able to see here is that Terraform is proposing to create two new resources: a VCN and a Subnet.  Both of these are expected and things appear to be in order, so we'll go ahead and apply it (tell Terraform to make the changes), by running:
-
-```console
-$ terraform apply
-```
-
-We'll see something like what we saw for plan, but have a prompt asking if we'd like to continue:
-
-```console
-<snip>
-Changes to Outputs:
-  + vcn_state = (known after apply)
-
-Do you want to perform these actions?
-  Terraform will perform the actions described above.
-  Only 'yes' will be accepted to approve.
-
-  Enter a value: 
-```
-
-Once we accept the proposed changes, we'll see something like:
-
-```console
-<snip>
-Do you want to perform these actions?
-  Terraform will perform the actions described above.
-  Only 'yes' will be accepted to approve.
-
-  Enter a value: yes
-
-oci_core_vcn.tf_101: Creating...
-oci_core_vcn.tf_101: Creation complete after 2s [id=ocid1.vcn.oc1.phx.<sanitized>]
-oci_core_subnet.vlan1: Creating...
-oci_core_subnet.vlan1: Creation complete after 2s [id=ocid1.subnet.oc1.phx.<sanitized>]
-
-Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
-
-Outputs:
-
-vcn_state = "AVAILABLE"
-$ 
-```
-
-Wow, that was easy!  One command to set up multiple resources... terrific!  
+At this point, Terraform is ready for us to give it directions on what OCI resources we want it to manage.  In Terraform terms, this is called a *plan*.  
+
+### A closer look at Terraform's proposed plan
+
+Let's look at the plan that Terraform proposes. The `terraform plan` command provides a preview of the actions that Terraform will take to configure resources according to our configuration file.
+
+1. In a console, run:
+
+   ```console
+   terraform plan
+   ```
+
+     Terraform echoes something similar to the following:
```

Comment:
The following grafs and code blocks are double indented, should only have one level of indentation